rdt/__init__.py

```diff
@@ -20,15 +20,13 @@
 
 
 def get_demo(num_rows=5):
-    """Generate random demo data with multiple data types.
+    """Generate demo data with multiple data types.
+
+    The first five rows are hard coded. The rest is randomly generated.
```

Comment:
```  
    # Randomly generate the remaining rows
    np.random.seed(42)
```  
I think we should say something like: The rest is randomly generated using ``np.random.seed(42)``.